EIPS/eip-7594.md

```diff
@@ -0,0 +1,52 @@
+---
+eip: 7594
+title: PeerDAS - Peer Data Availability Sampling
+description: Introducing simple DAS utilizing gossip distribution and peer requests
```

Comment:
```suggestion
description: Introducing one dimensional DAS utilizing peer gossip, distribution and requests
```

Reply:
allows to be more specific for this proposal and in future add a new EIPs for e.g. 2 dimensional peer das